### DISCUSSION TITLE
Max epochs

### DISCUSSION THREAD

**Discussion by hminle** [Jeanselme/NeuralFineGray#5] [General]
Hi @Jeanselme ,

I see in your examples, you set `max_epochs = 1000` .
For your paper, did you really train for 1000 epochs?
I tried to train your Neural FineGray for 1000 epochs on SEER dataset, and it's still running after several days.

Thanks

**Comment by Jeanselme** [comment]
Yes, the training on this dataset takes a couple of days (can be even slower if you do not use GPU) - check if you have the lastest version of the code that should fasten it for CPU use

**Reply by hminle** [reply]
Thank Vincent.
Yes, I used GPU and pulled your latest code. 
Look like I have to train each method per GPU

**Reply by Jeanselme** [reply]
Is the GPU usage good ? or do you see some issues in deploying it on your GPU ?

**Reply by hminle** [reply]
Thank Vincent, I have no issues with your code running on GPU, that's really good
